content/sensu-core/0.29/reference/filters.md

```diff
@@ -295,6 +296,135 @@ value), and the event data does not contain a matching [eval token
 attribute][13], an log entry indicating an error called `"filter eval unmatched
 tokens"` will be published to the Sensu server log._
 
+## Built-in Filters
+### Occurrences filter {#built-in-filters-occurrences}
+
+The occurrences filter is included in every install of Sensu.
+It lets you control the number of duplicate events that reach the handler.
+You can apply the occurrences filter to a handler using the `filters` handler definition attribute.
+
+Here's an example of a handler definition that uses the occurrences filter:
+
+{{< highlight json >}}
+{
+  "handlers": {
+    "email": {
+      "type": "pipe",
+      "command": "email.rb",
+      "filters": ["occurrences"]
+    }
+  }
+}
+{{< /highlight >}}
+
+#### Occurrences filter attributes {#occurrences-filter-attributes}
+
+The occurrences filter lets you configure the number of occurrences and the refresh interval at the check level using two check definition attributes: `occurrences` and `refresh`.
+
+Here's an example of a check definition that passes events to the `email` handler starting with the third occurrence every 60 minutes:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check-http": {
+      "command": "check-http.rb -u https://localhost:8080/api/v1/health",
+      "subscribers": ["web_application"],
+      "interval": 20,
+      "handlers": ["email"],
+      "occurrences": 2,
+      "refresh": 3600
+    }
+  }
+}
+{{< /highlight >}}
+
+occurrences  | 
+-------------|------
+description  | The number of events that must occur before an event is handled for the check
+required     | True if the specified handler uses the occurrences filter
+type         | Integer
+default      | 1
+example      | {{< highlight shell >}}"occurrences": 2{{< /highlight >}}
+
+refresh      | 
+-------------|------
+description  | Time in seconds until event occurrences are handled for the check again
+required     | False
+type         | Integer
+default      | 1800
+example      | {{< highlight shell >}}"refresh": 3600{{< /highlight >}}
+
+### Check dependencies filter {#check-dependencies-filter}
+
+The check dependencies filter is included in every install of Sensu.
+This filter can be applied to a handler using the `filters` handler definition attribute.
+The check dependencies filter lets you reduce notification noise by matching events when an event already exists, thereby only notifying for the root cause of a given failure.
```

Comment:
This part I think needs changed: `by matching events when an event already exists`.
The point of dependencies is to get to the root cause quicker. So if a tomcat process isn't running, a check that tries to get to the API for an application inside of tomcat would not work. API depends on tomcat, if tomcat down only handle that event.

Maybe something like `by looking to see if an event higher up in the dependency tree already exists`